tests/Paramore.Brighter.AWS.Tests/MessagingGateway/When_posting_a_message_via_the_messaging_gateway.cs

```diff
@@ -61,7 +61,7 @@ public SqsMessageProducerSendTests()
         public async Task When_posting_a_message_via_the_producer()
         {
             //arrange
-            _messageProducer.Send(_message);
```

Comment:
We should probably have one version of the test that calls the sync method, and one version that calls the async method.